journal/modernized/03-chapter-2.adoc

```diff
@@ -92,7 +92,7 @@ a sweet harmony of love and life attended us,
 and we were comforted in the Lord and one another.
 We had good service at that meeting in helping Friends in discipline;
 and the meeting ended in the love of God.
-Tidings having reached us of the death of John ap John`'s wife,
+Tidings having reached us of the death of __John ap John`'s__ wife,
```

Comment:
My gut is to remove the italics here and leave the name as it was. It's a little confusing the first time you see this name, but I sort of feel like we should either do nothing, or add a footnote, rather than italics, so my thought is to just put this back for now, if that seems good to you.